city/schemas.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseModel
+
+
+class CityCreate(BaseModel):
+    name: str
+    additional_info: str
+
+
```

Comment:
The field 'city_name' in the 'Temperature' schema should be 'city_id' to match the task description, which specifies that 'Temperature' should reference the city by 'city_id'.